src/util/colors.d.ts

```diff
@@ -0,0 +1,49 @@
+declare module 'vuetify/es5/util/colors' {
```

Comment:
Everything in here should be marked [`readonly`](https://www.typescriptlang.org/docs/handbook/interfaces.html#readonly-properties). You might have to export another interface to make that possible.